common/app/assets/assets.scala

```diff
@@ -39,6 +39,34 @@ class Assets(base: String, mapResource: String, useHashedBundles: Boolean = Conf
 
 }
 
+// turns a readable CSS class into a list of rules in short form from the atomic css file
+class CssMap(mapResource: String, useCssMap: Boolean = Configuration.assets.useHashedBundles) extends Logging {
+
+  lazy val lookup: Map[String, List[String]] = Get(cssMap(mapResource))
+
+  def apply(path: String): String = {
+    if (useCssMap) {
```

Comment:
Good point, I agree that special parameters only used for testing is not ideal - and I hadn't thought about greppability.  What I was going for was making the classes as pure as possible so that the tests don't need to know anything about the internals (composition over inheritance).
An overridable method for fencing off the behaviour not needed in the tests is a good way in such a simple class, although arguably adds more complexity than a defaulted parameter.

Assets/AssetsTest also use the same pattern so I'll look at that too - https://github.com/guardian/frontend/blob/master/common/test/assets/AssetsTest.scala#L9